R-package/R/lgb.model.dt.tree.R

```diff
@@ -1,9 +1,9 @@
 #' @name lgb.model.dt.tree
 #' @title Parse a LightGBM model json dump
 #' @description Parse a LightGBM model json dump into a \code{data.table} structure.
-#' @param model object of class \code{lgb.Booster}
-#' @param num_iteration number of iterations you want to predict with. NULL or
-#'                      <= 0 means use best iteration
+#' @param model object of class \code{lgb.Booster}.
+#' @param num_iteration Number of iterations to include. NULL or <= 0 means use best iteration.
+#' @param start_iteration Start index of the iteration (default 1).
```

Comment:
```suggestion
#' @param start_iteration Index (1-based) of the first boosting round to include in the output.
                    For example, passing \code{start_iteration=5, num_iteration=3} for a regression model
                    means "return information about the fifth, sixth, and seventh trees".
```

For anyone who didn't understand the parameter name `start_iteration`, I think this description as-is which re-uses the word "start" won't help their understanding.

Would you consider this alternative wording? And the same change across all of the `start_iteration` docs in roxygen comments introduced in this PR.